src/components/thumbnail-card/__tests__/ThumbnailCardDetails.test.js

```diff
@@ -25,4 +25,11 @@ describe('components/thumbnail-card/ThumbnailCardDetails', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should render actionItem if it is passed in', () => {
+        const actionItem = <div>Action Item</div>;
+        const wrapper = getWrapper({ actionItem });
+
+        expect(wrapper).toMatchSnapshot();
```

Comment:
please specify more specifically what this test expects to be rendered, instead of a snapshot